src/gen/relationalai/protocol/message_pb.jl

```diff
@@ -72,3 +72,8 @@ function PB._encoded_size(x::MetadataInfo)
     !isempty(x.relations) && (encoded_size += PB._encoded_size(x.relations, 1))
     return encoded_size
 end
+
+# Pretty printer to prevent large amounts of Metadata from being printed.
+function Base.show(io::IO, x::MetadataInfo)
+    print(io, "MetadataInfo (length=", length(x.relations), ")")
+end
```

Comment:
Following this, you might _also_ want to do a 2-arg show, for a shorter constructor syntax, which leaves off the type params.

.... Sadly, though, all of this is a bit moot, because the JSON that it's embedded in doesn't implement a valid julia 2-arg show. I filed an issue about that here: https://github.com/quinnj/JSON3.jl/issues/202.

I guess a user might still want to copy/paste just the Metadata portion of a response, so it would still be nice to support this, but it matters much less.
Also, I'm not sure if ProtoBuf objects can be constructed from their 2-arg show. That might have been fixed in the 1.0 rewrite, but we would have to check.

So anyway, it would probably be fine to add just a 3-arg show and leave off a shorter 2-arg show, or even to leave things how you have it in this PR, but i'd prefer to at least use 3-arg show here to be good citizens.